testing/cloud-utils/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: "Vladimir Avinkin <mouzfun@gmail.com>"
+# Maintainer: "Vladimir Avinkin <mouzfun@gmail.com>"
+pkgname=cloud-utils
+pkgver=0.30
+pkgrel=0
+pkgdesc="This package provides a useful set of utilities for interacting with a cloud"
+url="https://launchpad.net/cloud-utils/"
+arch="noarch"
+license="GPL-3.0"
+depends="sfdisk qemu-img"
+makedepends=""
+install=""
+subpackages="$pkgname-doc"
+source="https://launchpad.net/$pkgname/trunk/0.3/+download/$pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+check() {
+	cd "$builddir"
+	PATH=$PATH:$builddir/bin; for i in test/*; do sudo $i; done
```

Comment:
we should not run tests with sudo. we need figure out some other way to run tests.

Reply:
Well, it requires root in the check itself, i can skip checks alltogether. Not sure what else can be done here, those checks setup block device file, mount them etc.